spec/models/reports/region_summary_schema_spec.rb

```diff
@@ -322,6 +322,45 @@ def refresh_views
       end
     end
 
+    describe "#visited_without_bs_taken_rates" do
```

Comment:
Slightly unrelated to this PR: We could add a test for the cases where we saw a `0%` in the visits graphs for really small numbers.

Reply:
right, we could add it.

Reply:
Adding a test to check this behaviour in this function will make us create a lot of records. That will make the test suite really slow. So we have added this test to the rounded percentage function to test this behaviour.